data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+					accept
+				`	"Why do you seem so sad about the situation?"`
+					goto sad
+				`	"What will I receive in exchange for helping you?"`
+				`	"Why can't you locate it yourself?"`
+					goto locate
+	
+			`	"We will pay you <payment> if you are able to at least find where the Star Queen is, but there may be more reward if you are able to return it."`
+				goto end
+			
+			label sad
+			`	The man pauses before answering you, as if trying to choose his words carefully. "I personally know a lot of the scientists on board that Star Queen, and I just know that someone's going to wind up dead in this situation, and it isn't every day that someone you know gets held captive by pirates. This whole thing is just stressful for me."`
+				goto end
+	
+			label locate
+			`	"The Star Queen had heavy escorts that shouldn't have been easy to destroy, but seeing as how they were we need someone to forge ahead and find the Star Queen now while we assemble a proper rescue fleet. In a situation like this, every minute matters, and if someone like you can find the Star Queen fast then that means that there is a higher chance everyone can come out of this alive."`
+	
+			label end
+			choice
+				`	"Okay, I'm willing to help."`
+					accept
+				`	"Sorry, but this doesn't sound worth it to me."`
+					decline
+
+	on enter Arneb
+		set "found star queen"
+		dialog "As you enter Arneb, you spot the Star Queen landing on Haven, defended by a number of pirate vessels. You will need to report back to the Deep in order to retrieve the Star Queen in one piece."
+	
+	on visit
+		dialog "You have landed on <planet>, but you have not located the Star Queen! Find it somewhere in the Far North."
+	
+	to complete
+		has "found star queen"
+	
+	on complete
+		payment 100000
+
+mission "Bactrian Unlock [0]: Prime"
+	landing
+	source Prime
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`The Star Queen is not in this system, so you attempt to find somebody who may have some information. You eventually find a merchant who claims to have been here when the Star Queen was and that a large pirate fleet jumped in and took the Star Queen hostage before jumping north to Rigel.`
+				decline
+
+mission "Bactrian Unlock [0]: Farpoint"
+	landing
+	source Farpoint
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`After speaking with a Navy Officer for a few minutes, you find out that the Star Queen came through here over a week ago, escorted by a few large pirate ships. They did not want to risk the lives of those on the Star Queen by attacking the pirates, so they let the ship go and it continued north from here.`
+			`	There have been preparations for a rescue mission, but the fleet that originally captured the Star Queen had also destroyed a number of Navy capital ships, so they have been waiting for supplies before going out on the potentially dangerous mission.`
```

Comment:
Some other possibilities for this bit:
- "... a number of well-armed Navy ships, ..."  
- "... a number of heavily-armed Navy ships, ..." 
- "... several Navy Cruisers, ..." (or other ship models as appropriate).